robot/client/client.go

```diff
@@ -130,13 +128,6 @@ func skipConnectionCheck(method string) bool {
 	return exemptFromConnectionCheck[method]
 }
 
-func isClosedPipeError(err error) bool {
```

Comment:
Spoke with @dgottlieb a bit offline. We were relying on `io.ErrClosedPipe` to tell us if a client was disconnected in any of our gRPC calls within `robot/client/client.go`. We now explicitly rely on `rpc.ErrDisconnected`, which should be returned any time a gRPC call tries to open a new stream on a closed channel (we send a gRPC request over a closed connection.)